tornado-api/src/main/java/uk/ac/manchester/tornado/api/collections/types/Float6.java

```diff
@@ -175,15 +168,15 @@ public Float6 duplicate() {
     }
 
     public String toString(String fmt) {
-        return format(fmt, getS0(), getS1(), getS2(), getS3(), getS4(), getS5());
+        return String.format(fmt, getS0(), getS1(), getS2(), getS3(), getS4(), getS5());
     }
 
     @Override
     public String toString() {
-        return toString(fmt6);
+        return toString(FloatOps.fmt6);
     }
 
-    public static final Float6 loadFromArray(final float[] array, int index) {
+    public static Float6 loadFromArray(final float[] array, int index) {
```

Comment:
This change is consistent across all classes. I have this question:

If the return value is a final Object, then why do we remove it?

Reply:
We don't need the final since it is a `static` method.